src/Sarif.WorkItems/SarifWorkItemModel.cs

```diff
@@ -83,9 +83,14 @@ private string CreateBugFooter()
             if(this.Context.CurrentProvider == FilingClient.SourceControlProvider.AzureDevOps)
             {
                 StringBuilder azureDevOpsFooter = new StringBuilder();
-                azureDevOpsFooter.Append(@"To see result details, please visit the Scans tab of this bug, or the attached SARIF log.");
+                azureDevOpsFooter.Append(@"<br><br>Other viewing options:<br>");
                 azureDevOpsFooter.AppendLine();
-                azureDevOpsFooter.Append(@"If the scans tab is missing or unavailable, please install the SARIF viewer from https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer");
+                azureDevOpsFooter.AppendLine();
+                azureDevOpsFooter.Append(@"<li>Examine the complete <a href=""https://dev.azure.com/office/Office/_componentGovernance/Office?_a=alerts&typeId=1731351&alerts-view-option=active"">CG Scan</a> for this repository.</li>");
+                azureDevOpsFooter.AppendLine();
```

Comment:
So not only does this mention CG, but it mentions a _specific scan_ :) Is there actually a URL you can usefully put here?

Reply:
I'm...not sure.  This is in the template example, but I would need the Sarif log to contain this URI.  A cursory search by me did not find it, but if it exists then that should be what goes here.

---
In reply to: [398187964](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398187964) [](ancestors = 398187964)

Reply:
This is now moot with your new resource strings.

---
In reply to: [398191142](https://github.com/microsoft/sarif-sdk/pull/1823#discussion_r398191142) [](ancestors = 398191142,398187964)